content/en/metrics/distributions.md

```diff
@@ -14,26 +14,69 @@ further_reading:
+However, you can additionally enable the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` as well as advanced query options like threshold queries for SLOs.
 
 {{< img src="metrics/distributions/percentiles.mp4" alt="Enable Percentiles" video=true style="width:80%;">}}
 
 After electing to apply percentile aggregations on a distribution metric, these aggregations are automatically available in the graphing UI:
 
 {{< img src="metrics/distributions/graph_percentiles.jpg" alt="Distribution metric aggregations" style="width:80%;">}}
 
+You can use percentile aggregations in a variety of other widgets and for alerting: 
+* **Single percentile value over any timeframe**
+_“What has the p95 request duration for my application been over the past week?”_ 
+
+{{< img src="metrics/distributions/percentile_QVW.jpg" alt="Single Percentile Value QVW" style="width:80%;">}}
+
+* **Percentile thresholds on metric monitors**
+_ “Alert me when the p99 of my application’s request latency is greater than 200 ms for the last 5 min.”_ 
+
+{{< img src="metrics/distributions/percentile_monitor.jpg" alt="Percentile Threshold for Metric Monitors" style="width:80%;">}}
+
+### Threshold Queries for SLOs (beta) 
+<div class="alert alert-warning">
+Threshold queries are in beta. 
+</div>
+
+Enabling DDSketch-calculated globally-accurate percentiles on your distribution metrics unlocks threshold queries for SLOs where you can count the number of raw distribution metric values if they exceed or fall below a numerical threshold. So for example, you can define SLOs for the past 30 days where the p95 request latency is < 10 seconds. 
+
+With threshold queries for distributions with percentiles, you do not need to predefine a threshold value prior to metric submission, and have full flexibility to adjust the threshold value in Datadog.
+
+To use threshold queries: 
+
+1. Enable percentiles on your distribution metric on the Metrics Summary page.
+2. Graph any distribution metric with the “count values...” aggregator and specify a threshold value and comparison operator as shown here:
+
+{{< img src="metrics/distributions/threshold_queries.mp4" alt="Threshold Queries" style="width:80%;">}}
+
+Example metric-based SLO set up:
+
+{{< img src="metrics/distributions/threshold_SLO.jpg" alt="Threshold Queries for SLOs" style="width:80%;">}}
```

Comment:
This bit is super inaccessible, not sure we even need the image. If we're keeping it, can we crop it so the side bar on the right with all the extra text is gone, and add more text description to the steps so that we're actually describing this, not just showing an image?